src/utils/communitySbt/getSubgraphBadges.ts

```diff
@@ -0,0 +1,70 @@
+import { BadgeResponse } from "../../entities";
+import { ApolloClient, gql, HttpLink, InMemoryCache } from '@apollo/client';
+import { SubgraphBadgeResponse } from "../../entities/communitySbt";
+import { toMillis } from "./helpers";
+
+export async function getSubgraphBadges({
```

Comment:
feels like this needs testing?

Reply:
yes, it does. but for new, as a reassurance, this is a copy of the deleted bits from `entities/communitySbt/getSeasonBadges()`